src/sidebar/test/local-storage-test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var angular = require('angular');
+
+describe('sidebar.localStorage', () => {
```

Comment:
I might need to get the hang of using arrow functions. Is there a guideline you follow for when to use them vs `function`?